src/main/java/org/jeasy/props/processors/AbstractAnnotationProcessor.java

```diff
@@ -77,14 +77,14 @@ protected void rejectIfEmpty(final String value, final String message) throws An
      * @return the resource as {@link InputStream}
      * @throws IOException when an error occurs during resource loading
      */
-    protected InputStream getResourceAsStream(final String resource) throws IOException {
+    InputStream getResourceAsStream(final String resource) throws IOException {
```

Comment:
I think this can still as `protected`. Why do we need to make it package private?